Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -2198,7 +2200,105 @@ public struct Data : Equatable, Hashable, RandomAccessCollection, MutableCollect
     public func withUnsafeBytes<ResultType>(_ body: (UnsafeRawBufferPointer) throws -> ResultType) rethrows -> ResultType {
         return try _representation.withUnsafeBytes(body)
     }
-    
+
+#if compiler(>=6.2)
```

Comment:
I think we only support building this with the 6.2 compiler in the first place.

Reply:
I'll remove those. With an update to the README too.